client-memory.html

```diff
@@ -39,67 +56,95 @@
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
 
-</div>
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li><p class="first">Within Xcode, click and open the <span class="bold">JVMDefaultOptions</span> node.</p>
+
+<p>Under <span class="bold">Xmx</span>, you will see a value such as:&nbsp;&nbsp;&nbsp;&nbsp;<span class="code-example">-Xmx1024M</span>&nbsp;&nbsp;&nbsp;&nbsp;indicating you currently have 1024 Megabytes reserved for the application to use.</p>
+
+<p>Increase this to something like <span class="code-example">-Xmx2048M</span> if you wish to allocate 2048 Megabytes of memory.</p>
+
+<p>Viewing the file in a text editor, line <span class="bold">54</span>, has the value set as:&nbsp;&nbsp;&nbsp;&nbsp;<span class="code-example">&lt;string&gt;-Xmx1024M&lt;/string&gt;</span> .</p>
+
+<p>Edit the value to:&nbsp;&nbsp;&nbsp;&nbsp;<span class="code-example">&lt;string&gt;-Xmx2048M&lt;/string&gt;</span>&nbsp;&nbsp;&nbsp;&nbsp;to increase the allocated memory to 2048 Megabytes.</p>
+
+<p>Save the file and close.</p>
+
+<p>Restart OMERO.insight to load the new setting.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/clientMemoryEdit.jpg" class="align-center" src="images/clientMemoryEdit.jpg" style="width: 100%;" />
 
 <div class="line-block">
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
-</div><!-- End section -->
+</li>
+
+</ol>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
 
 <p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
 
-<div class="section" id="omero-importer-omero-insight-on-windows">
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
 
 <h2><a class="anchor" id="windows"></a>OMERO.insight on Windows</h2>
 
 <p>To alter the memory allocation of the Windows clients, you must add an optional parameter file to the same folder as the application.</p>
 
-<p>Create a file called <span class="bold">OMERO.insight.vmoptions</span> in the same directory as your <span class="bold">OMERO.insight.exe</span> file and insert a value such as <span class="bold">-Xmx2048M</span>, as described in<span class="bold"> OMERO.insight on OS X</span> above.</p>
- 
-</div><!-- End section -->
+<p>Create a file called <span class="code-example">OMERO.insight.vmoptions</span> in the same directory as your <span class="code-example">OMERO.insight_64.exe</span> or <span class="code-example">OMERO.insight.exe</span> file and insert a value such as <span class="code-example">-Xmx2048M</span>, as described in OMERO.insight on OS X above.</p>
```

Comment:
I wonder if "…and insert the text -Xmx2048M as the only content. This will allocate 2048 Megabytes of memory as described in the OMERO.insight on OS X section above." would be clearer